I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 275, in script
    $fechaS = suceso.strip()
  File "game/script.rpy", line 275, in <module>
    $fechaS = suceso.strip()
NameError: name 'suceso' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 275, in script
    $fechaS = suceso.strip()
  File "C:\Fabian\Programas\Renpy\renpy\ast.py", line 881, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Fabian\Programas\Renpy\renpy\python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 275, in <module>
    $fechaS = suceso.strip()
NameError: name 'suceso' is not defined

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
Novela_Visual 1.0
Thu Jul 04 20:04:42 2019
